Documentation/cli-installation-scenarios.md

```diff
@@ -0,0 +1,216 @@
+
+# General principles
+
+- Upgrades using the native installers Just Work(tm)
+- All user facing materials point to the getting started page
+- Defaults are stable bits; users need extra effort to install nightly builds
+- Only HTTPS links are allowed in any online property 
+- Provide native installers for each supported platform
+- Provide automation-ready installers for each target platform
+
+# Channels
+Channels represent a way for users who are getting the CLI to reason about the stability and quality of the bits they are getting. This is one more way for the user to be fully aware of the state the bits that are being installed are in and to set proper expectations on first use. 
+
+The table below outlines the channels:
+
+| Property         	| Description                                                                                                                                                                                                                                                       	|
+|------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| Nightly              	| Unstable bits that are "bleeding edge". Users are not expected to use this channel often, however it is there for those situations when someone needs/wants a feature that hasn't been stabilizied yet. Also, used for internal testing. 	|
+| Preview 	| Pre-release stable builds with known issues and/or known feature gaps. We are OK with users installing these for preview purposes.                                                                                                                                	|
+| Production          	| Actual releases. Most users are encouraged to install these.                                                                                                                                                                                                      	|
+
+Below table shows the mapping between the channels, branches and feeds for the Debian pacakage. Since channels also impact the NuGet packages, it also contains the version of the package used and the feed to opt in to each channel.
+
+| Channel    	| Branch    	| Debian feed 	| Debian package name 	| NuGet version 	| NuGet feed                            	|
+|------------	|-----------	|-------------	|---------------------	|---------------	|---------------------------------------	|
+| Nightly    	| master    	| Development 	| dotnet-nightly      	| 1.0.0-dev-*   	| https://dotnet.myget.org/f/dotnet-cli 	|
```

Comment:
Is everyone ok with this channel name or should it be dev/development? @brthor @Sridhar-MS @piotrpMSFT @livarcocc @blackdwarf